libpod/container_internal.go

```diff
@@ -681,6 +697,32 @@ func (c *Container) stop(timeout uint) error {
 	return c.cleanup()
 }
 
+// Internal, non-locking function to pause a container
```

Comment:
This is a nice change, but I would have preferred it being in a separate PR.  A) to make the PR review easier, B) so you can search for the pause/unpause change more easily in git many months hence.

Reply:
I can split the changes out trivially, this was just the first thing written that required an internal pause/unpause.